File: Lab 3/factorial.c
// C program to find factorial of given number
// Factorial using recursion
#include<stdio.h>
long int factorial(int n)  //Function to find factorial of given number
{
    if(n==0)
    return 1;  //base condition to stop the iteration
    else
    return n*factorial(n-1);  // Recursive Call
}

//Driver Code
int main()
{
    int n;
    printf("Enter the number: \n");
    scanf("%d",&n);
    printf("The factorial of %d is %ld\n",n,factorial(n));
    return 0;
}

File: Lab 3/factorial.py
# Python program to find factorial of given number

# Function to find factorial of given number
def factorial(n):
 if(n==0):
    return 1    # Base case to stop the iteration
 else:
    return n*factorial(n-1)   # Recursive Call

# Driver Code
num=int(input("Enter a number: "))
print("The factorial of", num ,"is",factorial(num))
